### PR TITLE
Fixed sticky sessions and added sticky session settings

### LINKED ISSUE #13: Sticky label is not working as expected.
Hi @dariopb , setting the sticky label within the SF application has no impact on the load balancer configuration. I got it working locally by initiating the internal Cookie struct within the Sticky struct. I will raise a PR to fix this issue along with implementing the additional Sticky labels, i.e., Name, HttpOnly, Secure, and SameSite.

Hi @ldez, one thing I noticed when configuring the Sticky session is that it's not displaying the fact that a service is configured with Sticky session within the dashboard. Is this expected behaviour? Note that I have validated that the cookie is being set within Postman, I've also confirmed that NTLM Auth is working which requires sticky sessions so I know it's turned on under the hood.

### PR DESCRIPTION
**Pull request by nokjuh**
Fixed #13 by initializing Cookie section of Sticky struct.

Added configuration settings for sticky sessions: Secure, HTTPOnly, SameSite, cookie name.

### CODE REVIEW

File: labels.go [hunk 1]
```diff
@@ -57,11 +57,60 @@ func setLoadbalancerSticky(lb *dynamic.ServersLoadBalancer, val string) error {
...
+		lb.Sticky.Cookie.HTTPOnly = v
+	}
+	return nil
+}
+
+func setLoadbalancerStickySameSite(lb *dynamic.ServersLoadBalancer, val string) error {
+	if lb.Sticky == nil {
+		lb.Sticky = &dynamic.Sticky{ Cookie: &dynamic.Cookie{} }
+	}
+	lb.Sticky.Cookie.SameSite = val
```
Comment (Mahons) [inline labels.go]: @nokjuh should we be defensive here and validate the string to ensure it's one of: 'none', 'lax', or 'strict'?

You'll need to **import "strings"** for the following suggestion to work.

```suggestion
	valid := map[string]bool{"none": true, "lax": true, "strict": true}

       if valid[strings.ToLower(val)] {
	     lb.Sticky.Cookie.SameSite = val
	} else {
	     log.Printf("Unrecognised value '%s' provided for Cookie.SameSite",  val)
             return nil
	}

```

File: labels.go [hunk 2]
```diff
@@ -57,11 +57,60 @@ func setLoadbalancerSticky(lb *dynamic.ServersLoadBalancer, val string) error {
 	}
 
 	if v {
-		lb.Sticky = &dynamic.Sticky{}
+		if lb.Sticky == nil {
+			lb.Sticky = &dynamic.Sticky{ Cookie: &dynamic.Cookie{} }
```
Comment (Mahons) [inline labels.go]: @nokjuh might be cleaner to create a function to initialise the struct and reuse it throughout the sticky related functions.
```go

func NewSticky() *Sticky {
    return &dynamic.Sticky{ Cookie: &dynamic.Cookie{} }
}

lb.Sticky = *NewSticky()
```

**Review by Mahons — CHANGES_REQUESTED** [review]
Nice job @nokjuh

File: labels.go (hunk 1 above)
Comment (nokjuh) [reply labels.go]: This is now done.

File: labels.go (hunk 2 above)
Comment (nokjuh) [reply labels.go]: This is now done.

**Review by dariopb — APPROVED** [review]

**Pull request merged**